src/shared/pages/team/teamView/teamView.tsx

```diff
@@ -42,121 +44,170 @@ const onEditTeamAddClick = (history: History, teamId: string) => {
+                        <p>
+                            {hasRole('ACTIVATE_TEAM') &&
+                            <Link to={`/team/${teamId}/reactivate`} type="submit"
+                                className="govuk-link">Reactivate team</Link>}
+                        </p>
+                    </p>}
+
+                    <h2 className="govuk-heading-m govuk">
+                        <div>{`Unit: ${state.unitName}`}</div>
+                    </h2>
+                    <h3 className="govuk-heading-l">View and remove team members</h3>
+
+                    {
+                        state.teamMembersLoaded ?
+                            <div>
+                                <DisplayTeamTable/>
+                            </div> :
+                            <div>
+                                <p className="govuk-body">Loading...</p>
+                            </div>
+                    }
+                    <button type="submit" className="govuk-button govuk-!-margin-right-1 add-team-members-button"
+                        data-module="govuk-button"
+                        onClick={() => onAddTeamMembersAddClick(history, teamId as string)}>Add team members
+                    </button>
+                    <button type="submit"
+                        className="govuk-button govuk-!-margin-right-1 govuk-button--secondary manage-nominated-contacts-button"
+                        data-module="govuk-button"
+                        onClick={() => onAddNominatedContactClick(history, teamId as string)}>Manage nominated
+                        contacts
+                    </button>
+                    {hasOneOfRoles(['RENAME_TEAM', 'REASSIGN_TEAM_UNIT']) &&
+                    <button type="submit" className="govuk-button govuk-button--secondary govuk-!-margin-right-1"
+                        data-module="govuk-button" onClick={() => onEditTeamAddClick(history, teamId)}>Edit
+                        Team</button>}
+                    {(state.active && hasRole('DEACTIVATE_TEAM'))
+                    && <button onClick={() => onDeactivateTeamAddClick(history, teamId)} type="submit"
+                        className="govuk-button govuk-button--secondary">Deactivate team</button>}
+                    {(!state.active && hasRole('ACTIVATE_TEAM'))
+                    && <button onClick={() => onReactivateTeamAddClick(history, teamId)} type="submit"
+                        className="govuk-button govuk-button--secondary">Reactivate team</button>}
+                </div>
+            </div>
+        );
+    };
+
 const WrappedTeamView = (routeProps: RouteComponentProps<MatchParams>) => (
     <ApplicationConsumer>
         {
-            ({ hasOneOfRoles }: ApplicationState) => {
-                return <TeamView {...routeProps} hasOneOfRoles={hasOneOfRoles}/>;
+            ({ hasOneOfRoles, hasRole }: ApplicationState) => {
+                return <TeamView {...routeProps} hasOneOfRoles={hasOneOfRoles} hasRole={hasRole}/>;
```

Comment:
Interestingly, there appears to be a `hasOneOfRoles` that we may be able to use instead of passing this `hasRole`? Not sure how it fits together...